docs/software/social.md

```diff
@@ -28,3 +24,227 @@
+
+=== "Access to the web interface"
+
+    - URL = `<http://<your.IP>/freshrss`
+    - Username = `dietpi`
+    - Password = `dietpi`
+
+=== "Usage"
+
+    Please refer to the official online documentation for usage information:  
+    <https://freshrss.github.io/FreshRSS/en/users/02_First_steps.html>
+
+## phpBB - Free flat-forum bulletin board software solution
+
+If you always wanted your very own forum, phpBB is all you need.
+
+Also Installs:
+
+- Webserver
+
+![DietPi social software phpBB](../assets/images/dietpi-software-social-phpbb.png){: style="width:500px"}
+
+### Access Forums
+
+URL = `http://<your.IP>/phpBB3`  
+(from DietPi v6.33 on: `http://<your.IP>/phpbb`)
+
+### First Run Setup
+
+DietPi automatically creates the SQL database for phpBB. Please follow the steps below to enter those details into phpBB to complete the setup.
+
+#### Access to the phpBB website
+
+- URL = `http://<your.IP>/phpBB3`  
+  (from DietPi v6.33 on: `http://<your.IP>/phpbb`)
+- Click the "Install" tab
+- Click "Proceed to next step"
+- Click "Start install"
+
+#### MySQL/MariaDB database details
+
+To enter the MySQL/MariaDB database details:
+
+- Database server hostname or data source name (DSN) = `localhost`
+- Database username = `phpbb3` (from DietPi v6.33 on: `phpbb`)
+- Database name = `phpbb3` (from DietPi v6.33 on: `phpbb`)
+- Database password = `dietpi` (respectively your chosen global software password)
+
+- Click "Proceed to next step"
+- Click "Proceed to next step"
+
+#### Create your admin account
+
+This account will be used for full access to phpBB forum.
+
+- Click "Proceed to next step"
+- Click "Proceed to next step"
+- Click "Proceed to next step"
+
+#### Email and Server URL settings
+
+Use the default values.
+
+- Click "Proceed to next step"
+- Click "Proceed to next step"
+
+The database tables will now be generated, please wait.
+
+#### Login
+
+Click "Login"  
+Using your admin login details that you created previously, you can now manage and create your forum pages.
+
+As the installation is now completed, you will need to remove the installation folder, before the forum can be live. Run the following command:
+
+```sh
+rm -R /var/www/phpBB3/install
+# from DietPi v6.33 on:
+rm -R /var/www/phpbb/install
+```
+
+Your forum is now ready.
+
+## Wordpress - Website Blog and Publishing platform
+
+WordPress is a state-of-the-art semantic personal publishing platform with a focus on aesthetics, web standards, and usability.
+It allows you to create your own website.
+
+![DietPi social software Wordpress](../assets/images/dietpi-software-social-wordpress.jpg){: style="width:500px"}
+
+=== "Access to the web interface"
+
+    URL = `http://<your.IP>/wordpress`
+
+=== "First time connect"
+
+    The first time you connect, enter the following MySQL database details:
+
+    - Database name = `wordpress`
+    - Username = `wordpress` (Previous to DietPi v6.0 = `root`)
```

Comment:
Let's not add ancient info anymore here and remove `(Previous to DietPi v6.0 = `root`)`.

Reply:
Ok. Deleted.